orangecontrib/text/util.py

```diff
@@ -45,3 +50,44 @@ def wrapper(self, data, chunk_number=100, on_progress=None, *args, **kwargs):
         return res
 
     return wrapper
+
+
+def create_corpus(documents, name, attributes, class_vars, metas):
```

Comment:
Is this general enough to be somewhere on Corpus, like `Corpus.from_documents(list, ...)`? NLTK and Gensim have similar `from_documents()`.